lib/experimental/fonts/font-library/font-library.php

```diff
@@ -57,6 +57,24 @@ function wp_register_font_collection( $config ) {
 	}
 }
 
+if ( ! function_exists( 'wp_unregister_font_collection' ) ) {
+	/**
+	 * Unregister Font Collection from the Font Library.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $collection_id The font collection's id.
```

Comment:
```suggestion
	 * @param string $collection_id The font collection ID.
```
- Also removed `'s` for consistency with the previous function. Could also be kept here and added to the previous function if that's preferred.